layouts/footer.html

```diff
@@ -1,3 +1,3 @@
 </div>
-<div id="footer"><p>Copyright &copy; 2007-2013 The OpenRA Developers (<a href="/legal/">Legal notices</a>)</p></div>
-</div>
\ No newline at end of file
+<div id="footer"><p>Copyright &copy; 2007-2017 The OpenRA Developers (<a href="/legal/">Legal notices</a>)</p></div>
```

Comment:
Note that this will still only get updated when nanoc builds the website, not when people visit it. Still better than having to update it manually each year, though.